mhkit/wave/io/wecsim.py

```diff
@@ -0,0 +1,422 @@
+            forceRestoring.append(bodies[0][0]['forceRestoring'][0][body])
+            forceMorrisonAndViscous.append(bodies[0][0]['forceMorrisonAndViscous'][0][body])
+            forceLinearDamping.append(bodies[0][0]['forceLinearDamping'][0][body])    
+    except:
+        num_bodies = 0         
+        
+    ######################################
+    ## create body_output DataFrame
+    ######################################            
+    if num_bodies == 1:
+        body_output = pd.DataFrame(data = time[0],columns=['time'])   
+        body_output = body_output.set_index('time') 
+        body_output.name = name
+        for body in range(num_bodies):
+            for dof in range(6):
+                body_output[f'position_dof{dof+1}'] = position[body][:,dof]
+                body_output[f'velocity_dof{dof+1}'] = velocity[body][:,dof]
+                body_output[f'acceleration_dof{dof+1}'] = acceleration[body][:,dof]            
+                body_output[f'forceTotal_dof{dof+1}'] = forceTotal[body][:,dof]
+                body_output[f'forceExcitation_dof{dof+1}'] = forceExcitation[body][:,dof]
+                body_output[f'forceRadiationDamping_dof{dof+1}'] = forceRadiationDamping[body][:,dof]
+                body_output[f'forceAddedMass_dof{dof+1}'] = forceAddedMass[body][:,dof]
+                body_output[f'forceRestoring_dof{dof+1}'] = forceRestoring[body][:,dof]
+                body_output[f'forceMorrisonAndViscous_dof{dof+1}'] = forceMorrisonAndViscous[body][:,dof]
+                body_output[f'forceLinearDamping_dof{dof+1}'] = forceLinearDamping[body][:,dof]              
+    elif num_bodies > 1:
+        body_num_output = {}          
+        for body in range(num_bodies):
+            tmp2 = pd.DataFrame(data = time[0],columns=['time'])   
+            tmp2 = tmp2.set_index('time') 
+            tmp2.name = name[body]
+            for dof in range(6):                
+                tmp2[f'position_dof{dof+1}'] = position[body][:,dof]
+                tmp2[f'velocity_dof{dof+1}'] = velocity[body][:,dof]
+                tmp2[f'acceleration_dof{dof+1}'] = acceleration[body][:,dof]            
+                tmp2[f'forceTotal_dof{dof+1}'] = forceTotal[body][:,dof]
+                tmp2[f'forceExcitation_dof{dof+1}'] = forceExcitation[body][:,dof]
+                tmp2[f'forceRadiationDamping_dof{dof+1}'] = forceRadiationDamping[body][:,dof]
+                tmp2[f'forceAddedMass_dof{dof+1}'] = forceAddedMass[body][:,dof]
+                tmp2[f'forceRestoring_dof{dof+1}'] = forceRestoring[body][:,dof]
+                tmp2[f'forceMorrisonAndViscous_dof{dof+1}'] = forceMorrisonAndViscous[body][:,dof]
+                tmp2[f'forceLinearDamping_dof{dof+1}'] = forceLinearDamping[body][:,dof]                            
+            body_num_output[f'body{body+1}'] = tmp2            
+        body_output = body_num_output.copy()               
```

Comment:
Kelly a common theme I see and would suggest we improve is repeated code insider a for-loop and split by an if  for a single instance vs multiple instances. To improve readability and decrease the potential for bugs e.g. only changing one line in the future or fat fingering something we do not have a test, for now, I believe we need to create a sub-function for the common parts.

Reply:
I merged with master, resolved the path issue, and the build should be passing now https://travis-ci.org/github/MHKiT-Software/MHKiT-Python/builds/729140582. 

I'll work on creating sub-functions for single vs multiple instances now.

Reply:
@ssolson, the [build is passing](https://travis-ci.org/github/MHKiT-Software/MHKiT-Python/jobs/729152635), and the [coverage ](https://travis-ci.org/github/MHKiT-Software/MHKiT-Python/jobs/729152635)is up to 91%.  Let me know if there is anything else you could like me to change. If not, I'll work on converting the example to a Jupyter notebook.